src/applications/facility-locator/utils/helpers.js

```diff
@@ -80,6 +80,42 @@ export const areGeocodeEqual = (pos1, pos2) => {
   return (pos1.latitude === pos2.latitude) && (pos1.longitude === pos2.longitude);
 };
 
+/**
+ * Compares two geographic bounding boxes to determine if they are equal.
+ *
+ * A bounding box is expected to be of the shape
+ *   [lat1, long1, lat2, long2]
+ *
+ * @param {number[]} box1 The first bounding box's coords
+ * @param {number[]} box2 The second bounding box's coords
+ */
+export const areBoundsEqual = (box1, box2) => {
```

Comment:
Could we get a unit test for this too?

It looks like it's not being used right now, but as long as it's "live" (not commented out), it should probably be tested.

Reply:
Yep, should be easy enough to add that in. Pretty sure `helpers.js` already has a unit test suite.